worlds/celeste64/Options.py

```diff
@@ -142,7 +238,68 @@ class Celeste64Options(PerGameCommonOptions):
     friendsanity: Friendsanity
     signsanity: Signsanity
     carsanity: Carsanity
+    checkpointsanity: Checkpointsanity
+
+    madeline_one_dash_hair_color: MadelineOneDashHairColor
+    madeline_two_dash_hair_color: MadelineTwoDashHairColor
+    madeline_no_dash_hair_color: MadelineNoDashHairColor
+    madeline_feather_hair_color: MadelineFeatherHairColor
 
     badeline_chaser_source: BadelineChaserSource
     badeline_chaser_frequency: BadelineChaserFrequency
     badeline_chaser_speed: BadelineChaserSpeed
+
+
+def resolve_options(world: World):
+    # One Dash Hair
+    if isinstance(world.options.madeline_one_dash_hair_color.value, str):
+        try:
+            world.madeline_one_dash_hair_color = int(world.options.madeline_one_dash_hair_color.value.strip("#")[:6], 16)
+        except ValueError:
+            raise OptionError(f"Invalid input for option `madeline_one_dash_hair_color`:"
+                              f"{world.options.madeline_one_dash_hair_color.value} for "
+                              f"{world.multiworld.get_player_name(world.player)}")
```

Comment:
You can just use this
```suggestion
                              f"{world.player_name}")
```